SevenZip/ArchiveExtractCallback.cs

```diff
@@ -564,6 +564,11 @@ private static void ValidateFileNameAndCreateDirectory(string fileName)
         /// <returns></returns>
         private static string RemoveIllegalCharacters(string str, bool isDirectory = false)
         {
+#if NET462
+            var isUncPath = str.StartsWith(@"\\?\");
```

Comment:
I would make `@"\\?\"` a variable that can then be accesses in the other places in this function.

Reply:
fixed